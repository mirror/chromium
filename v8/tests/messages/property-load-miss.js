function A() { this.foo = 4 }
function get_foo(obj) { return obj.foo; }
get_foo(new A());
get_foo(null);
