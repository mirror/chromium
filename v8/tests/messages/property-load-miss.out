property-load-miss.js:2: Uncaught TypeError: Cannot read property 'foo' of null
function get_foo(obj) { return obj.foo; }
                                  ^
